jdaviz/configs/imviz/wcs_utils.py

```diff
@@ -204,7 +204,7 @@ def draw_compass_mpl(image, orig_shape=None, wcs=None, show=True, zoom_limits=No
 
     fig, ax = plt.subplots()
     ax.imshow(image, extent=[-0.5, orig_shape[1] - 0.5, -0.5, orig_shape[0] - 0.5],
-              origin='lower', cmap='gray', **kwargs)
+              origin='lower', cmap='gray', transform=transform + ax.transData, **kwargs)

```

Comment:
This is raising
```python
--> 207           origin='lower', cmap='gray', transform=transform + ax.transData, **kwargs)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'CompositeGenericTransform'
```
when opening the Compass without any rotation applied. Rotating first and then opening the viewer looks correct!

Reply:
I am guessing your suggested commit fixed this? Thanks!

Reply:
Yes, and you also need to pull the latest update from the `glue` PR.

Reply:
I'll put this in my queue for next week. Thx!

Reply:
... and apply the second suggestion to change parameters to
```python
origin='lower', cmap='gray', transform=transform, **kwargs)
```
(again, why does GH not just allow one to put everything into one suggestion...? 🤯)

Reply:
I don't see any second suggestion.